app-editors/bluefish/bluefish-2.2.8.ebuild

```diff
@@ -1,11 +1,11 @@
-# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Also, I suppose it would be better to drop whichever versions can be dropped isntead of fixing them all.